vaultfs/auth.go

```diff
@@ -349,6 +352,72 @@ func (m *userPassAuthMethod) Logout(ctx context.Context, client *api.Client) err
 	return revokeToken(ctx, client)
 }
 
+// KubernetesAuthMethod authenticates to Vault with the kubernetes auth method. If
+// role is omitted, the value will be read from the $VAULT_AUTH_ROLE. If jwtPath is
+// omitted we will attempt to read from $VAULT_AUTH_SATOKEN_PATH environment variable or
```

Comment:
Actually I wonder if `VAULT_AUTH_KUBERNETES_TOKEN_PATH` would be clearer:

```suggestion
// omitted we will attempt to read from $VAULT_AUTH_KUBERNETES_TOKEN_PATH environment variable or
```

This follows somewhat the same naming scheme as other variables - e.g. `VAULT_AUTH_GITHUB_TOKEN`...